lib/jobs/isc-dhcp-lease-poller.js

```diff
@@ -84,14 +84,16 @@ function iscDhcpLeasePollerFactory(
 
         Promise.try(function () {
             var lease = self.parseLeaseData(data.toString());
-
-            self.queue.enqueue(
-                lookupService.setIpAddress.bind(
-                    lookupService,
-                    lease.ip,
-                    lease.mac
-                )
-            );
+            
+            if (!_.isUndefined(lease)) {
+                self.queue.enqueue(
+                    lookupService.setIpAddress.bind(
+                        lookupService,
+                        lease.ip,
+                        lease.mac
+                    )
+                );
+            }
```

Comment:
Good question.  I'm assuming there wasn't a new lease to look up and parseLeaseData returns undefined.